libraries/botbuilder-ai/src/luisAdaptiveRecognizer.ts

```diff
@@ -105,6 +105,10 @@ export class LuisAdaptiveRecognizer extends Recognizer implements LuisAdaptiveRe
         '=settings.runtimeSettings.telemetry.logPersonalInformation'
     );
 
+    /**
+     * @param property properties that extends RecognizerConfiguration
```

Comment:
Check the comments added to use uppercase and end them with a dot.
```suggestion
     * @param property Properties that extend RecognizerConfiguration.
```